client/my-sites/sidebar/sidebar.jsx

```diff
@@ -384,17 +387,28 @@ export class MySitesSidebar extends Component {
 			site,
 			siteSuffix,
 			translate,
+			transferStatus,
+			hasSitePendingAT,
 		} = this.props;
+
+		if ( ! config.isEnabled( 'woocommerce/extension-dashboard' ) || ! site ) {
+			return null;
+		}
+
 		const storeLink = '/store' + siteSuffix;
-		const showStoreLink =
-			config.isEnabled( 'woocommerce/extension-dashboard' ) &&
-			site &&
+
+		const isJetpackOrAtomicSite =
 			isJetpack &&
 			canUserManageOptions &&
 			( config.isEnabled( 'woocommerce/store-on-non-atomic-sites' ) ||
 				this.props.isSiteAutomatedTransfer );
 
-		if ( ! showStoreLink ) {
+		const siteHasBackgroundTransfer = hasSitePendingAT && transferStatus !== transferStates.ERROR;
```

Comment:
You could compute the `siteHasBackgroundTransfer` in `mapStateToProps` and pass one prop instead of two. I believe it simplifies things a little bit.

Reply:
Fixed in 06d5136.